ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -53,19 +53,119 @@ interface Management1
+!
+ipv6 access-list TEST2
+   counters per-entry
+   5 permit ipv6 2001:db8::/64 any
+   10 deny ipv6 2001:db8::/32 any
+!
+ipv6 access-list TEST3
+   5 deny ipv6 2001:db8:1000::/64 any
+   10 permit ipv6 2001:db8::/32 any
+!
+ipv6 access-list standard TEST4
+   5 deny fe80::/64
+   10 permit fe90::/64
+!
+ipv6 access-list standard TEST5
+   counters per-entry
+   5 permit 2001:db8::/64
+   10 deny 2001:db8::/32
+!
+ipv6 access-list standard TEST6
+   5 deny 2001:db8:1000::/64
+   10 permit 2001:db8::/32
+!
+ip access-list 4
+   10 remark ACL to restrict access RFC1918 addresses
+   20 deny ip 10.0.0.0/8 any
+   30 permit ip 192.0.2.0/24 any
+!
+ip access-list ACL-01
+   10 remark ACL to restrict access to switch API to CVP and Ansible
+   20 deny ip host 192.0.2.1 any
+   30 permit ip 192.0.2.0/24 any
+!
+ip access-list ACL-02
+   counters per-entry
+   10 remark ACL to restrict access RFC1918 addresses
+   20 permit ip 10.0.0.0/8 any
+   30 permit ip 192.0.2.0/24 any
+   permit response traffic nat
+!
+ip access-list ACL-03
+   10 remark ACL to restrict access RFC1918 addresses
+   20 deny ip 10.0.0.0/8 any
+   30 permit ip 192.0.2.0/24 any
+!
+ip access-list ACL-04
+   counters per-entry
+   20 deny ip 12.0.0.0/8 any
+   30 permit ip 194.0.2.0/24 any
+   permit response traffic nat
+!
+ip access-list ACL_NO_SEQUENCE
+   remark test acl without sequence numbers
+   deny udp any any log
+   permit icmp any any 3 4 ttl eq 40
+   permit icmp any any unreachable ttl gt 3
+   permit ip any any fragments dscp 46
+   permit ip any any tracked dscp ef
+   permit ip any any nexthop-group NH_TEST
+   permit vlan inner 123 0x000 ip any any
+   permit vlan 234 0xFFF ip any any
+   permit icmp any any
+!
+ip access-list ACL_SEQUENCE_AND_COUNTERS
+   counters per-entry
+   10 remark test acl with sequence numbers
+   20 permit ip 10.0.0.0/8 any
+   30 permit tcp host 192.168.122.22 any established
+   40 permit tcp any gt 1023 host 172.16.16.16 eq 22
+   50 permit tcp any range 1000 1100 any range 10 20
+   4294967295 deny ip any any
+   permit response traffic nat
+!
+ip access-list standard 99
+   10 remark ACL to restrict access RFC1918 addresses
+   20 permit 10.0.0.0/8
+   30 permit 172.16.0.0/12
+   40 permit 192.168.0.0/16
+!
+ip access-list standard ACL-API
+   10 remark ACL to restrict access to switch API to CVP and Ansible
+   20 permit host 10.10.10.10
+   30 permit host 10.10.10.11
+   40 permit host 10.10.10.12
+!
+ip access-list standard ACL-SSH
+   counters per-entry
+   10 remark ACL to restrict access RFC1918 addresses
+   20 permit 10.0.0.0/8
+   30 permit 172.16.0.0/12
+   40 permit 192.168.0.0/16
+!
+ip access-list standard ACL-SSH-VRF
+   10 remark ACL to restrict access RFC1918 addresses
+   20 permit 10.0.0.0/8
+   30 permit 172.16.0.0/12
+   40 permit 192.168.0.0/16
+!
 ip nat pool prefix_16 prefix-length 16
-   range 10.0.0.1 10.0.255.254
-   range 10.1.0.0 10.1.255.255 1024 65535
+   range 10.0.0.1 10.0.0.1
```

Comment:
Getting error when IPs are different

s1-leaf1(config-natpool-prefix_32)#range 10.1.0.1 10.1.1.1
% Start and End Ip addresses do not lie in same subnet.

Reply:
Only one ip contains in  /32 subnet.

Reply:
fixed